app/code/Magento/Quote/Model/Quote/Address/Total.php

```diff
@@ -54,6 +55,9 @@ public function __construct(
      */
     public function setTotalAmount($code, $amount)
     {
+        /* (Fixes issue #18027) Round the total amount to 4 decimal places, to avoid floating point overflows */
```

Comment:
These comments are not necessary for final codebase, please remove them from both functions